resources/xcode_settings.bash

```diff
@@ -5,7 +5,7 @@ set -e
 defaults write com.apple.dt.Xcode DVTTextEditorTrimTrailingWhitespace -bool YES
 defaults write com.apple.dt.Xcode DVTTextEditorTrimWhitespaceOnlyLines -bool YES
 
-defaults write com.apple.dt.Xcode DVTTextIndentTabWidth -int 2
-defaults write com.apple.dt.Xcode DVTTextIndentWidth -int 2
+defaults write com.apple.dt.Xcode DVTTextIndentTabWidth -int 4
+defaults write com.apple.dt.Xcode DVTTextIndentWidth -int 4
 
-defaults write com.apple.dt.Xcode DVTTextPageGuideLocation -int 100
+defaults write com.apple.dt.Xcode DVTTextPageGuideLocation -int 120
```

Comment:
This is cool but how does this actually get pulled in for use? Should we link to our dev environment setup script?

If that's too much work for now, we should probably just omit this script.